messenger.rst

```diff
@@ -729,7 +736,7 @@ factories. Defining such requires a two-step configuration based on Symfony's
 
         // Step 2: an abstract definition that will call the factory with default
         // arguments or the ones provided in the middleware config
-        $container->register('messenger.middleware.doctrine_transaction_middleware', DoctrineTransactionMiddleware::class)
+        $container->register('messenger.middleware.doctrine_transaction', DoctrineTransactionMiddleware::class)
             ->setFactory(array(
                 new Reference('doctrine.orm.messenger.middleware_factory.transaction'),
```

Comment:
Hasn't the factory been removed?

Reply:
Not in 4.1

Reply:
I thought it was part of DoctrineBundle which doesn't care whether you use the Messenger component in version 4.1 or 4.2. Isn't that the case?

Reply:
The factory is in the symfony bridge actually